packages/jest-runtime/src/transform.js

```diff
@@ -26,7 +26,6 @@ const VERSION = require('../package.json').version;
 
 type Options = {
   isInternalModule?: boolean,
-  instrument?: boolean,
```

Comment:
i don't remember why i added this option. seems like it's not used anywhere and none of the tests fail if i remove it